detectron2/evaluation/evaluator.py

```diff
@@ -123,10 +124,12 @@ def inference_on_dataset(model, data_loader, evaluator):
                 torch.cuda.synchronize()
             total_compute_time += time.perf_counter() - start_compute_time
             evaluator.process(inputs, outputs)
+            total_time += time.perf_counter() - start_compute_time
 
             if idx >= num_warmup * 2:
                 seconds_per_img = total_compute_time / (idx + 1 - num_warmup)
-                eta = datetime.timedelta(seconds=int(seconds_per_img * (total - idx - 1)))
+                total_seconds_per_img = total_time / (idx + 1 - num_warmup)
```

Comment:
Why not just 
```
 total_seconds_per_img = (time.perf_counter() - start_time) / (idx + 1 - num_warmup)
```
it will then also include data time (and simpler). If you aim for more accurate ETA, data time should be included as well.

Reply:
done